test/e2e/nightwatch.json

```diff
@@ -5,7 +5,7 @@
   "output_folder": "reports",
   "custom_commands_path": "",
   "custom_assertions_path": "",
-  "page_objects_path": "",
+  "page_objects_path": "test/e2e/pageObjects",
```

Comment:
cool...this matches what I was thinking...but I would like to put all page objects under an adminUI subdirectory just in case keystone implements some other sorts of UI later on.  what you think?